src/components/singleTransactionV2/singleTransactionV2.js

```diff
@@ -146,7 +146,7 @@ class SingleTransactionV2 extends React.Component {
             </footer>
           </main>
         </BoxV2>
-      ) : (
+      ) : this.props.transaction && typeof this.props.transaction === 'string' && (
```

Comment:
you just need to use` transaction && typeof transaction === 'string' `as you already use destructuring in the top